src/cards/cards.css

```diff
@@ -1,7 +1,20 @@
 /* https://docs.google.com/presentation/d/1ORi0ZjFeOoNyK7l3R8ms9v0uSmLerUIUfG1fXh_XcTk/edit#slide=id.g5b283a7c15_0_86 */
+@media only screen and (min-width: 481px) {
+  .omrs-card {
+    background-color: #ffffff;
+    border-radius: 0.5rem;
+    border: 1px solid #dadce0;
+    padding: 3rem;
```

Comment:
Also I don't think cards should have padding, because the padding varies depending on the specific feature.

Reply:
@joeldenning the padding is specified in the figma designs. What would be the alternative to achieve the same?